### PR TITLE
Fixes #15067: rudder-server-relay depends of apache2-mod_wsgi-python3 on sles which does not exist

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/15067

### CODE REVIEW

File: rudder-server-relay/SPECS/rudder-server-relay.spec [hunk 1]
```diff
@@ -93,7 +93,11 @@ Requires: python3, python3-mod_wsgi
 BuildRequires: python
 Requires: python, apache2-mod_wsgi, python-pyOpenSSL
 %endif
-%if 0%{?suse_version} && 0%{?suse_version} >= 1500
+%if 0%{?suse_version} && 0%{?suse_version} == 1500
+BuildRequires: python
+Requires: python, apache2-mod_wsgi-python
```
Comment (peckpeck) [inline rudder-server-relay/SPECS/rudder-server-relay.spec]: do you mean we will use python2

File: rudder-server-relay/SPECS/rudder-server-relay.spec [hunk 2]
```diff
@@ -93,7 +93,11 @@ Requires: python3, python3-mod_wsgi
...
+%endif
+%if 0%{?suse_version} && 0%{?suse_version} > 1500
```
Comment (amousset) [inline rudder-server-relay/SPECS/rudder-server-relay.spec]: Which OS is this targeting?

Comment (peckpeck) [inline rudder-server-relay/SPECS/rudder-server-relay.spec]: this does not exist yet and we cannot presupose what it will contain

**Comment by Fdall** [comment]
Commit modified

File: rudder-server-relay/SPECS/rudder-server-relay.spec (hunk 1 above)
Comment (Fdall) [reply rudder-server-relay/SPECS/rudder-server-relay.spec]: yes, there is no apache2-mod_wsgi-python3 support on sles15

**Comment by Fdall** [comment]
Closing this since apache2-mod_wsgi is not provided in the official base repos.
We will ditribute the python3 version.

**Pull request closed without merging**